tensorflow/compiler/xla/service/gpu/ir_emission_utils.cc

```diff
@@ -312,17 +312,70 @@ llvm::Value* EmitPrintf(absl::string_view fmt,
        arguments_ptr});
 }
 
+// Helper function to emit call to AMDGPU shfl_down function.
```

Comment:
Rather than taking a vector of operands, take two arguments with descriptive names.